app/uk/gov/hmrc/selfservicetimetopay/models/package.scala

```diff
@@ -73,41 +74,26 @@ package object modelsFormat {
 }
 
 package object controllerVariables {
-  implicit val paymentSchedules = Seq(
-    CalculatorPaymentSchedule(None, None, new BigDecimal("200.00"), new BigDecimal("2000.00"), new BigDecimal("1800.00"),
-      new BigDecimal("20.00"), new BigDecimal("2020.00"), Seq(
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 1, 1), new BigDecimal("300.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 2, 1), new BigDecimal("300.00"))
-      )
-    ),
-    CalculatorPaymentSchedule(None, None, new BigDecimal("200.00"), new BigDecimal("2000.00"), new BigDecimal("1800.00"),
-      new BigDecimal("20.00"), new BigDecimal("2020.00"), Seq(
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 1, 1), new BigDecimal("250.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 2, 1), new BigDecimal("250.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 3, 1), new BigDecimal("250.00"))
-      )
-    ),
-    CalculatorPaymentSchedule(None, None, new BigDecimal("200.00"), new BigDecimal("2000.00"), new BigDecimal("1800.00"),
-      new BigDecimal("20.00"), new BigDecimal("2020.00"), Seq(
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 1, 1), new BigDecimal("100.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 2, 1), new BigDecimal("100.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 3, 1), new BigDecimal("100.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 4, 1), new BigDecimal("100.00"))
-      )
-    ),
-    CalculatorPaymentSchedule(None, None, new BigDecimal("200.00"), new BigDecimal("2000.00"), new BigDecimal("1800.00"),
-      new BigDecimal("20.00"), new BigDecimal("2020.00"), Seq(
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 1, 1), new BigDecimal("50.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 2, 1), new BigDecimal("50.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 3, 1), new BigDecimal("50.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 4, 1), new BigDecimal("50.00")),
-        CalculatorPaymentScheduleInstalment(LocalDate.of(2016, 5, 1), new BigDecimal("50.00"))
-      )
-    )
-  )
+  def generatePaymentSchedules(amountDue:BigDecimal, payment:Option[BigDecimal]):Seq[CalculatorPaymentSchedule] = {
```

Comment:
I assume this isn't intended to be production code, and just something used to generate some values for demo purposes right now?